React/wbooks/src/redux/logout/actions.js

```diff
@@ -0,0 +1,14 @@
+import api from '../../Config/api';
+import history from '../../Config/history';
+import localStorage from '../../Services/LocalStorage';
+
+export const userActions = { logout: 'LOGOUT' };
+
+export default {
+  logout: () => async dispatch => {
```

Comment:
this doesn't need to be an async action. You can do:
```
logout: () => {
  localStorage.removeSessionToken();
  api.setHeader('Authorization', '');
  history.push('/login');
  return { type: userActions.logout }
}
```

If you ever have to perform a request to logout, then in that case you will have to make it async